node/node-pakiti2-json.py

```diff
@@ -37,9 +38,9 @@
     print(response.read())
 
   except:
-    print('[]')
+    print('Cannot Connect')
```

Comment:
The js does a `array_key_exists()` check on it. If the `except` is triggered, it will return false as it's just a string.